config/application.yml.default

```diff
@@ -496,7 +495,7 @@ production:
   s3_report_bucket_prefix: login-gov.reports
   s3_report_public_bucket_prefix: login-gov-pubdata
   s3_reports_enabled: true
-  saml_endpoint_configs: '[]'
+  saml_endpoint_configs: '[{"suffix":"2024","secret_key_passphrase":"trust-but-verify"},{"suffix":"2023","secret_key_passphrase":"trust-but-verify","comment":"this extra year is needed to demonstrate how handling multiple live years works in spec/requests/saml_requests_spec.rb"}]'
```

Comment:
[question] is this a testing artifact or something?

Reply:
Yes, it is needed only because the review app uses the `production` config. I have included it as a separate commit that will be dropped before the merge.